### ISSUE TITLE
[Alias] BBM sometimes 2blind

### ISSUE THREAD

**Issue by gkistler** [Clam-/pyBurlyBot#71]
CKX: When bbm joins a channel, he should iterate over the nick list and add them all to seen users. This way even if a channel member has never spoken before, bbm still has a record of them (and users can send .tells, etc).

```
(10:36:44) * buttasspoo (~qwebirc@Rizon-D89EC83F.hsd1.mi.comcast.net) has joined #BEFREEAN
(10:36:53) <Grau> .alias buttasspoo rona
(10:36:54) <bbm> (buttasspoo) not seen before.


<- :buttasspoo!~qwebirc@Rizon-D89EC83F.hsd1.mi.comcast.net JOIN :#BEFREEAN
-> irc.rizon.sexy PRIVMSG #BEFREEAN :.alias buttasspoo rona
<- :bbm!~bbm@big.burly.man PRIVMSG #BEFREEAN :(buttasspoo) not seen before.
```

Probably needs to hook userJoined and that's it.  I want to be able to fire bbm (.join the_worst_Channel) and have all; user states populate (.seen "last saw when I entered thischan" ) and I want to be able to make channels anymous via commands, pretty sure this is in already but just lmk

**Comment by Clam-** [comment]
I agree somewhat on this. bbm should "see" them, but he hasn't really seen them "speak."
So a user entry could/should probably be made, but there's no actual "seen" data I guess...
Will think about how to handle it while I do the alias group thing
